lib/chef/resource_inspector.rb

```diff
@@ -43,7 +43,6 @@ def self.extract_resource(resource, complete = false)
     data[:actions] = resource.allowed_actions
     data[:examples] = resource.examples
     data[:introduced] = resource.introduced
-    data[:preview] = resource.preview_resource
```

Comment:
We want to keep preview as a descriptor even if we don't use it to mean anything from the code side. It powers the docs and allows us to arbitrarily mark things as stable or not.